src/main/kotlin/net/atos/zac/smartdocuments/SmartDocumentsService.kt

```diff
@@ -46,57 +46,79 @@ class SmartDocumentsService @Inject constructor(
     fun listTemplates() = documentCreatieService.listTemplates().toREST()
 
     /**
-     * Stores template mapping for ZaakafhandelParameters
+     * Stores template mapping for zaakafhandelparameters and informatieobjecttypes
      *
      * @param restTemplateGroups a set of RESTSmartDocumentsTemplateGroup objects to store
-     * @param zaakafhandelUUID UUID of the ZaakafhandelParameters
+     * @param zaakafhandelUUID UUID of the zaakafhandelparameters
+     * @param informatieObjectTypeUUID UUID of the informatieobjecttype
      */
     @Transactional(REQUIRED)
-    fun storeTemplatesMapping(restTemplateGroups: Set<RESTSmartDocumentsTemplateGroup>, zaakafhandelUUID: UUID) {
+    fun storeTemplatesMapping(
+        restTemplateGroups: Set<RESTSmartDocumentsTemplateGroup>,
+        zaakafhandelUUID: UUID,
```

Comment:
`zaakafhandel` is not a thing.. ;-) I think it is confusing to use it like this. Also, personally I don't like to abbreviate terms like this. It often just causes confusion later on.

```suggestion
        zaakafhandelparametersUUID: UUID,
```